src/org/omegat/gui/glossary/GlossarySearcher.java

```diff
@@ -240,14 +251,21 @@ static void sortGlossaryEntries(List<GlossaryEntry> entries) {
             int p1 = o1.getPriority() ? 1 : 2;
             int p2 = o2.getPriority() ? 1 : 2;
             int c = p1 - p2;
+            if (c == 0 && Preferences.isPreferenceDefault(Preferences.GLOSSARY_SORT_BY_SRC_LENGTH, false)
```

Comment:
From discussion on the issue ticket, we can enable it in default.
```suggestion
            if (c == 0 && Preferences.isPreferenceDefault(Preferences.GLOSSARY_SORT_BY_SRC_LENGTH, true)
```